libclingo/src/control.cc

```diff
@@ -2196,6 +2201,29 @@ extern "C" bool clingo_control_register_observer(clingo_control_t *control, clin
     GRINGO_CLINGO_CATCH;
 }
 
+extern "C" bool clingo_control_register_backend(clingo_control_t *control, clingo_backend_type_t type, char const *file, bool replace) {
+    GRINGO_CLINGO_TRY { 
+        auto out = gringo_make_unique<std::ofstream>(file);
+        auto backend = UBackend{}; //Output::make_backend(out, OutputFormat::reify, false, false);
```

Comment:
Nitpick: consider removing the commented-out code.

Reply:
Fixed.